src/components/app/Home.js

```diff
@@ -370,6 +300,8 @@ class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
               type="button"
               className="homeSectionButton"
               onClick={this._enableMenuButton}
+              disabled={!webChannelAvailable}
+              title={webChannelAvailable ? undefined : 'WebChannel unavailable'}
```

Comment:
This should probably be localizd too. You can do that with an attribute, see this as an example: https://github.com/firefox-devtools/profiler/blob/d953efae95a94420ccbca3962328bafa9727d0ef/locales/en-US/app.ftl#L23-L24 https://github.com/firefox-devtools/profiler/blob/4336a8b8cab33af8c3c1832969afe821b0702cb3/src/components/app/AppHeader.js#L46

Reply:
And probably we should have a better sentence too

Reply:
Suggestion for the sentence:
```
We can't add the profiler button from this page because this profiler instance was unable to connect to the WebChannel.
```